site/content/docs/5.0/getting-started/rtl.md

```diff
@@ -174,7 +174,7 @@ Need both LTR and RTL on the same page? Thanks to [RTLCSS String Maps](https://r
       "scope": "*",
       "ignoreCase": false
     }
-  ]
+  ] }
```

Comment:
I think it is rather `} ]` as in https://github.com/twbs/bootstrap/commit/75fe3244db7cd99a08cd524fa379dcde8fafbced

Reply:
Yes my bad...